server/seeder.js

```diff
@@ -0,0 +1,31 @@
+const connection = require('./database.js');

```

Comment:
You called this `database` in another file, yet `connection` here. I would recommend sticking to a consistent variable naming scheme given that this is all code written by you.